### ISSUE TITLE
WindowsTerminal as Custom Shell not launching into project directory

### ISSUE THREAD

**Issue by Stanzilla** [ForkIssues/TrackerWin#475]
When using a "Windows Terminal" custom shell, the path is not switched properly and the shell opens at the root.

**Comment by DanPristupov** [comment]
> This is a "feature" of Windows Terminal.
> 
> You must change 'profiles.json':
> 
> `"startingDirectory" : ".",`
> 
> More info here: https://github.com/microsoft/terminal/issues/1060

P.S. The issue title should be more precise

**Comment by Stanzilla** [comment]
@DanPristupov yeah okay but how do I make Fork launch WT into the directory of the current git project?

**Comment by d1vanloon** [comment]
When the default Windows Terminal profile is configured with `"startingDirectory": "."`, the Terminal button in Fork launches Windows Terminal in the appropriate directory.

Verified with Fork `1.40.2.0` and Windows Terminal `0.5.2762.0`.

**Comment by Stanzilla** [comment]
Decent workaround but would it be possible to pass the starting dir as an argument anyway? I don't know much about the WT internals.

**Comment by d1vanloon** [comment]
@Stanzilla Unless the previously mentioned change is made in `profiles.json`, Windows Terminal will ignore any starting directory requests and does not accept a starting directory parameter. Discussion is ongoing: https://github.com/microsoft/terminal/issues/878

**Comment by andrewskirrow** [comment]
You can now set the working directory on the command line as follows

`wt new-tab -d c:\some\directory`

It'd be useful to be able to pass custom parameters to the terminal too so that a custom profile could be choosen etc.

**Comment by DanPristupov** [comment]
@andrewskirrow yes, this is planned for next update.

> `wt new-tab -d c:\some\directory`

Or `wt new-tab -d .` to use the working directory.

**Comment by DanPristupov** [comment]
1.46 is released. Could anybody confirm if everything work properly on your side?

**Comment by Stanzilla** [comment]
Can confirm!

**Comment by AndeYashwanth** [comment]
I put the ``` "startingDirectory" : "."``` in profiles.json and it is working well. 
But today I downloaded a folder which has [ ] in its name. When I type wt in windows explorer address bar the PowerShell directory is set to ```C:\Windows\System32\WindowsPowerShell\v1.0```.
Issue can be recreated by creating a folder and renaming it to ```[]``` or anything containing ```[``` and ```]```.
I found that it was strange and wanted to mention that. 
edit:
When I press tab after typing cd I just noticed the folder containing ```[]``` was autofilled containing ` before [ and ].
